pandas/core/util/hashing.py

```diff
@@ -62,11 +62,11 @@ def hash_pandas_object(
 
     Parameters
     ----------
-    index : boolean, default True
+    index : bool, default True
         include the index in the hash (if Series/DataFrame)
-    encoding : string, default 'utf8'
-        encoding for data & key when strings
-    hash_key : string key to encode, default to _default_hash_key
+    encoding : str, default 'utf8'
+        encoding for data & key when str
+    hash_key : str key to encode, default to _default_hash_key
```

Comment:
```suggestion
    hash_key : str, default '_default_hash_key'
```
And move the explanation to the description

Reply:
Not sure about "move the explanation to the description", Is not it already in the Parameters description area? Should I have to add a line to describe why it is done?
Sorry, if I'm asking something lame, newbie!